lib/omniauth-dsds/spec/sign_in_helper.rb

```diff
@@ -15,9 +15,9 @@ def mock_token(token: "ABCDEF")
           })
         end
 
-        def mock_profile(token: "ABCDEF", profile: {}, authentication_site_url: "http://app.example.com")
-
-          response_body = profile.reverse_merge({
+        def mock_profile(token: "ABCDEF", options: {}, authentication_site_url: "http://app.example.com")
+          payload = {}
+          response_body = payload.reverse_merge({
```

Comment:
Why call reverse_merge on an empty Hash?

May as well just assign the default hash directly to response_body